src/resolvers/Post/comments.ts

```diff
@@ -0,0 +1,8 @@
+import { PostResolvers } from "../../types/generatedGraphQLTypes";
+import { Comment } from "../../models";
+
+export const comments: PostResolvers["comments"] = async (parent) => {
+  return await Comment.find({
+    postId: parent._id,
```

Comment:
You know something like this makes sense to be populated with additional data like `user` who created the comment. A comment will mostly always be fetched with information about the user who created it, so it makes sense to include the data along with the comment.